src/main/java/wooteco/subway/exception/SubwayException.java

```diff
@@ -0,0 +1,22 @@
+package wooteco.subway.exception;
+
+import org.springframework.http.HttpStatus;
+import wooteco.subway.exception.dto.ErrorResponse;
+
+public class SubwayException extends RuntimeException {
+    HttpStatus httpStatus;
+    ErrorResponse errorResponse;
```

Comment:
```suggestion
    private final HttpStatus httpStatus;
    private final ErrorResponse errorResponse;
```

접근제어자를 생략하신 것은 의도된 내용일까요!?

Reply:
아니요 😅 깜빡했네요!